_i18n/en/documentation/bugs-first-aid/refresh-download-errors.md

```diff
@@ -0,0 +1,14 @@
+Do you get a notification saying that downloads completed with errors? Then AntennaPod could not refresh the podcast or download an audio file from the website of the podcast hoster.
+
+Issues that only occur with one or a few specific podcasts are often related to those podcasts. To confirm if this is the case, you can follow these steps:
+1. Tap on the notification to open the Download Log
+2. Tap on the specific log entry with the error
+3. Long-press on the URL that is displayed, and copy or open it in your web browser
+
+If the URL doesn't work in the browser either, this means that it is not a problem with AntennaPod. If you get the error message consistently then the podcast has likely been moved, taken off-line or is not properly configured. In this case, please contact the podcast creators (as this cannot be fixed at our end).
+
+Does the error message in the Download log say "Failed to connect to [url]"? Then this probably means that there is interference on your network. If you have an ad-blocker, VPN, captive portal, proxy, privacy guard, special DNS service, or parental control set up on your phone, you can try turning these off.
+
+Do you see an error message like "not well-formed (invalid token)”? Then the publisher messed up their (RSS) feed. (For example, they might have used a plain `&`-sign rather than encoding it as `&amp;`.) 
```

Comment:
```suggestion
Do you see an error message like "not well-formed (invalid token)”? Then the publisher messed up their (RSS) feed. (For example, they might have used a plain `&`-sign rather than encoding it as `&amp;`.) Please contact the publisher directly, as this cannot be fixed at our end.
```